setup.cfg

```diff
@@ -4,16 +4,16 @@ tag_build =
 [nosetests]
 with-sqla_testing = true
 where = test
-cover-package = ibm_db
```

Comment:
I'm not sure these are really right and since we're going to move to pytest in the future anyway, why don't we just remove these sections till we know what should be in here